cluster-autoscaler/cloudprovider/aws/aws_manager.go

```diff
@@ -247,6 +247,17 @@ func (m *AwsManager) buildNodeFromTemplate(asg *asg, template *asgTemplate) (*ap
 	node.Status.Capacity[gpu.ResourceNvidiaGPU] = *resource.NewQuantity(template.InstanceType.GPU, resource.DecimalSI)
 	node.Status.Capacity[apiv1.ResourceMemory] = *resource.NewQuantity(template.InstanceType.MemoryMb*1024*1024, resource.DecimalSI)
 
+	resourcesFromTags := extractResourceAllocatableFromAsg(template.Tags)
+	if val, ok := resourcesFromTags["cpu"]; ok {
```

Comment:
Do you need to override cpu and memory resource? I thought you only want `ephemeral-storage`  in this case?

Reply:
I don't think we can predict the value of `kubelet`s `--kube-reserved=` so I thought this would be a nice addition for those, who need to set the nodes to the exact value that is gonna be allocatable.
Should I remove it?

Reply:
It's true and kubelet can not predict the value, actually neither of the user, since there's kube-reserved and system-reserved value.  Do you meet any problems using node capacity now?

Reply:
no, we are quite new to the autoscaling game. We don't even set `--kube-reserved` currently.
It felt like something that could become an issue down the road, when we increase our cluster utilization. Shall I remove CPU and memory it then and just open another merge request should we ever face an issue with that?

Reply:
Sounds good. As I know I don't see anyone experience this issue. this is just used to get a similar node from cloud provider. If you do see some issues, please submit another one. Let's concentrate on `ephemeral-storage` issue in this PR.